manim/mobject/number_line.py

```diff
@@ -163,13 +157,6 @@ def __init__(
         numbers_to_include: Optional[Iterable[float]] = None,
         **kwargs,
     ):
-        # deprecation
-        number_scale_value = kwargs.pop("number_scale_value", None)
-        if number_scale_value is not None:
-            self.font_size = number_scale_value * DEFAULT_FONT_SIZE * 0.75
-        else:
-            self.font_size = font_size
```

Comment:
I think you might've forgotten to re-add the font_size parameter here.

Reply:
here it is now. I hope the location is good.